src/Database/Expression/CaseExpression.php

```diff
@@ -246,4 +281,26 @@ public function traverse(callable $visitor)
             $this->_elseValue->traverse($visitor);
         }
     }
+
+    /**
+     * Helper build a fluent case builder
+     * If Value is not an ExpressionInterface, it will be treated as a value
+     *
+     * @param string $partName property to append the part
+     * @param string|\Cake\Database\ExpressionInterface $value Boolean Condition or Case Value
+     * @param string $type value type
+     * @return $this
+     */
+    protected function _addPart($partName, $value, $type = null)
```

Comment:
Is this method still used?

Reply:
Sorry, not anymore, I'll work in this PR maybe on weekend.